data_initialisation.py

```diff
@@ -250,6 +291,9 @@ def clean_regulatory_elements(regulatory_elements):
 
 def merge_annotation_expression(gene_annotation_data, general_expression_data, specific_expression_data):
     
+    #Joins all previously created dataframes into one, alligned by the gene
```

Comment:
This should be made into a general function, not one bespoke to your project. All you are essentially doing is merging dataframes together. So, if you look into python ***args** - a feature that allows you to enter an unspecified amount of arguments into a function - then you should be able to make this into a general function, and then rename it to _merge_dataframes_, or something, accordingly.